project.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=68.1.2",
+            "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "GemGIS"
+authors = [
+    {name = "Alexander Jüstel", email = "alexander.juestel@rwth-aachen.de"},
+]
+description = "Spatial data processing for geomodeling"
+keywords = ["visualization", "python", "processing", "jupyter", "modeling", "geospatial", "jupyter-notebook",
```

Comment:
this is a nice list of keywords but you might think about these in terms of words that directly relate to your package's functionality to help users find it. i'm not sure if jupyter is one you wish to use unless you somehow are building a tool that integrates into jupyter? maybe you are - just be sure these are specific to your package as it helps people find you on pypi

Reply:
Adapted